src/jit/compiler.h

```diff
@@ -8279,6 +8279,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
             return compMinOptsIsSet;
         }
 #endif // !DEBUG
+
+        inline bool OptimizationDisabled()
```

Comment:
The inline specifier isn't needed on function definitions that are inside the class.

Reply:
Scrubbed these and moved the `LclVarDsc` constructor into the class.